scripts/test.sh

```diff
@@ -72,7 +72,7 @@ function iterate_dir() {
         # check if test_case/check.ts exists
         if [ -f "$test_case/check.ts" ]; then
             # run the test
-            NETWORK="$network" npx mocha -r ts-node/register "$test_case/check.ts" && success=$((success + 1)) || echo "Test failed!"
+            NETWORK="$network" node -r ts-node/register "$test_case/check.ts" && success=$((success + 1)) || echo "Test failed!"
```

Comment:
I think this can be `npx ts-node ...`

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/259/commits/c5648b478f26cef526334f74981f76d0ab238286